src/azure-cli/azure/cli/command_modules/security/commands.py

```diff
@@ -261,5 +285,32 @@ def load_command_table(self, _):
         g.custom_command('list', 'list_security_sub_assessments')
         g.custom_command('show', 'get_security_sub_assessment')
 
+    with self.command_group('security iot-solution',
+                            security_iot_solution_sdk,
+                            client_factory=cf_security_iot_solution) as g:
+        g.custom_command('list', 'list_security_iot_solution')
+        g.custom_command('show', 'get_security_iot_solution')
+        g.custom_command('create', 'create_security_iot_solution')
+        g.custom_command('delete', 'delete_security_iot_solution')
```

Comment:
could we also add confirmation=True for all delete commands?

Reply:
I can do that 
but it will not stay consistent with the rest of the delete commands
the SDK handles it and will return an error if needed

Reply:
adding confirmation will align with current command guideline. adding `-y` in command will by pass the prompt. so I will recommend to add confirmation in all delete command.